README.md

```diff
@@ -64,6 +64,7 @@ Versions lookup table:
 
 | `ipympl` | `jupyter-matplotlib` | `JupyterLab` | `Matplotlib` |
 |----------|----------------------|--------------|--------------|
+| 0.8.9    | 0.10.x               | 3 or 2       | 3.4.0>=      |
```

Comment:
I would prefer if we do version bumps separately, if you don't mind

Reply:
:+1: